src/containers/ThreadPage.js

```diff
@@ -25,29 +25,21 @@ const { array, bool, func, object } = React.PropTypes
 export default class ThreadPage extends React.Component {
   static propTypes = {
     post: object,
+    comments: array,
     error: object,
     dispatch: func
   }
 
-  static childContextTypes = {
-    post: object,
-    comments: array
-  }
-
   static contextTypes = {
     isMobile: bool,
     currentUser: object
   }
 
-  getChildContext () {
-    return pick(['post', 'comments'], this.props)
-  }
-
   render () {
-    const { post, error } = this.props
+    const { post, comments, error } = this.props
     if (error) return <AccessErrorMessage error={error} />
     if (!post || !post.user) return <div className='loading'>Loading...</div>
 
-    return <Thread post={post} />
+    return <Thread post={post} comments={comments} />
```

Comment:
not sure what this change is about -- I don't see corresponding changes in `Thread`.